manager-5.1 - Added migration steps for SL Micro 6.1 to Retail Branch Server 4.3 (bsc#1262168) (#4973)
uyuni-project/uyuni-docs: modules/retail/pages/migration-from-43.adoc

mgrctl api post system/provisionSystem '{\"sid\":$proxyid,\"profileName\":\"$profileName\"}'
----

=== Reinstall with {sl-micro} 6.1

{sl-micro} 6.1 requires a fresh installation on the host.

[IMPORTANT]
====

> Perform the backup of the existing 4.3 Branch Server as described in xref:#retail-backup-43[] before proceeding with the host reinstallation.

Author: ```suggestion
Perform the backup of the existing 4.3 Branch Server before proceeding with the host reinstallation.
For more information see xref:#proxy-backup-43[].
```
